### PR TITLE
Refactor mane_info() to avoid sprintf().  Does part of the work of …

### PR DESCRIPTION
**Pull request by backwardsEric**
…resolving https://github.com/hengband/hengband/issues/2858 and https://github.com/hengband/hengband/issues/2859 .

### CODE REVIEW

File: src/cmd-action/cmd-mane.cpp [hunk 1]
```diff
@@ -78,46 +78,36 @@ static int damage;
 
 /*!
  * @brief 受け取ったパラメータに応じてものまねの効果情報をまとめたフォーマットを返す
- * @param p 情報を返す文字列参照ポインタ
  * @param power ものまねの効力の種類
  * @param dam ものまねの威力
+ * @param std::string 電力に関する印刷可能なコメント付き
```
Comment (habu1010) [inline src/cmd-action/cmd-mane.cpp:83]: The natural Japanese expression is "ものまねの効果を表す文字列".

Comment (backwardsEric) [reply src/cmd-action/cmd-mane.cpp]: The forced push should have fixed the comment from the automatically translated result (Google Translate English->Japanese) to your suggestion.

**Comment by backwardsEric** [comment]
The latest forced push adds const to the declaration at cmd-action/cmd-mane.cpp:222.

**Review by Hourier — APPROVED** [review]
Thank you for reflecting our comments.
I checked the current codes and found no problem.
I approve this.

**Pull request merged**